API/Filters/ExceptionFilter.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using API.Errors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Filters
+{
+	public class ExceptionFilter : IAsyncExceptionFilter
+	{
+		public async Task OnExceptionAsync(ExceptionContext context)
+		{
+			Console.WriteLine(context.Exception.Message);
+			
+			var response = new ApiException(context.HttpContext.Response.StatusCode, "Internal server error");
+			
+			var options = new JsonSerializerOptions
```

Comment:
Generally the JsonSerializerOptions should be aligned in the whole soultion, so suggest to move this options setting to the Startup.cs as a global setting

Reply:
I tried to globally set the JSON option in startup.cs like:
            services
                .AddControllers(_config => {
                    _config.Filters.Add(new ExceptionFilter());
                })
                .AddJsonOptions(option => {
                    option.JsonSerializerOptions.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
                });

However, it didn't work, is there any mistake I made here?